jscomp/test/splice_test.ml

```diff
@@ -37,6 +37,25 @@ let dynamic arr =
 ;; dynamic [||]
 ;; dynamic [|1;1;3|]
 
+(* Array constructor with a single parameter `x`
+   just makes an array with its length set to `x`,
+   so at least two parameters are needed
+*)
+external newArr : int -> int -> int array -> int array = "Array"
```

Comment:
Does it test all there is to test, or are there other combinations one should think about?